### ISSUE TITLE
Note from CRAN that tests are failing with R 3.2.5 on Windows

### ISSUE THREAD

**Issue by sfirke** [sfirke/janitor#76]
Note from Uwe Ligges:

> Dear maintainer,
> 
> please see
> 
> https://cran.r-project.org/web/checks/check_results_janitor.html
> 
> and find that your package faild under R-oldrelease (R-3.2.5) with the packages available there. Please fix or declare proper version dependencies and resubmit.

Relevant part of [the log](https://www.r-project.org/nosvn/R.check/r-oldrel-windows-ix86+x86_64/janitor-00check.html):

```
checking tests ... ERROR
Running the tests in 'tests/testthat.R' failed.
Last 13 lines of output:
  `%>%`(...) not equal to "bad name".
  1/1 mismatches
  x[1]: "bad.name"
  y[1]: "bad name"


  testthat results ================================================================
  OK: 189 SKIPPED: 0 FAILED: 2
  1. Failure: bad input variable name is preserved (@test-crosstab.R#108) 
  2. Failure: bad input variable name is preserved (@test-tabyl.R#100) 

  Error: testthat unit tests failed
  Execution halted
```

While I am curious to know why this code does not work under R 3.2.5 for Windows, from April 2016, I don't think I should spend time looking into this.  Instead, I'll specify a dependency on R >= 3.3.0, which I verified works.  That was released May 2016, so hopefully there aren't many people who haven't updated since then who will be inconvenienced by this.

**Comment by sfirke** [comment]
My curiosity got the better of me and I installed R 3.2.5 for Windows (thanks CRAN for having a .exe and RStudio for making it easy to toggle back and forth).  Turns out `as.data.frame(..., check.names = FALSE)` was not working but `data.frame(..., check.names = FALSE)` did.  So not adding that dependency above after all.

Though I did run into trouble not having dplyr 0.5.0 in the old R environment, so I added that specification to Imports in the `DESCRIPTION` file.  Not 100% sure how that will play out so please do open an issue if it causes problems or confusion.

**Comment by sfirke** [comment]
Opening until I actually resubmit to CRAN

**Comment by sfirke** [comment]
fixed in version 0.2.1 now on CRAN!
